controller/tests/controller_server/utils_test.py

```diff
@@ -392,9 +402,9 @@ def _check_node_id_parameters(self, node_id_info, nvme_nqn, fc_wwns, iscsi_iqn):
         self.assertEqual(node_id_info.initiators.iscsi_iqn, iscsi_iqn)
 
     def test_get_node_id_info(self):
-        with self.assertRaises(array_errors.HostNotFoundError) as ex:
+        with self.assertRaises(ValueError) as ex:
             utils.get_node_id_info("bad-node-format")
-            self.assertTrue("node" in str(ex))
+        self.assertIn("node", str(ex.exception))
```

Comment:
does it fail without `.exception`?
if not, then I would remove it

Reply:
yes

Reply:
done

Reply:
I don't see any change in the line about which this conversation is

Reply:
done